test/integration/volumescheduling/volume_binding_test.go

```diff
@@ -926,8 +926,10 @@ func setupCluster(t *testing.T, nsName string, numberOfNodes int, resyncPeriod t
 
 func initPVController(context *testContext, provisionDelaySeconds int) (*persistentvolume.PersistentVolumeController, informers.SharedInformerFactory, error) {
 	clientset := context.clientSet
-	// Informers factory for controllers, we disable resync period for testing.
-	informerFactory := informers.NewSharedInformerFactory(clientset, 0)
+	// Informers factory for controllers
+	// Use a frequent resync period to retry API update conflicts due to
+	// https://github.com/kubernetes/kubernetes/issues/85320
+	informerFactory := informers.NewSharedInformerFactory(clientset, 5)
```

Comment:
5 nanoseconds? the default is 12 hours... disabling completely (as we currently do) is more representative of how the controllers will actually behave. I would worry that an abnormally short resync would mask real problems in how the controller requeues/reattempts things.

Reply:
good catch I meant 5 seconds and the controller resync, not informer resync